src/components/molecules/Card/components/CardImage.tsx

```diff
@@ -19,6 +20,10 @@ export interface CardImageProps extends DefaultComponentProps {
   imageMargin?: boolean
   linkUrl?: string
   type?: CardHeaderType
+  tag?: string
+  contentTypeIconStyle?: React.CSSProperties | undefined
+  contentTypeIcon?: IconName
+  tagStyle?: React.CSSProperties | undefined
```

Comment:
No need to specify the `undefined` if you already have an `?`:

```suggestion
  contentTypeIconStyle?: React.CSSProperties
  contentTypeIcon?: IconName
  tagStyle?: React.CSSProperties
```

And why adding the word `Type` in `contentTypeIcon` and `contentTypeIconStyle`?